features/integration-github-bitbucket-gitlab.md

```diff
@@ -14,3 +14,59 @@ You can then either create a new organization directly synchronized with a GitHu
+* Synchronization:
+    * Keeps teams, their members, and access permissions in sync with your GitHub organisation
+    * The default repository permissions on Github is used to grant members access to your repositories
+* Code Credentials: GitHub App or GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### GitHub Enterprise
+* OAuth: Users authenticate on Private Packagist with their GitHub accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/github-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps team members and access permissions in sync with your GitHub Enterprise organisations
+    * The default repository permissions on Github is used to grant members access to your repositories
+* Code Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### Bitbucket Cloud (bitbucket.org)
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts
+* Synchronization:
+    * Keeps groups, their members, and access permissions in sync with your Bitbucket team
+* Code Credentials: Bitbucket API Key or Bitbucket App Password
+* Webhooks: Code changes and releases
+
+#### Bitbucket Server / Stash
+* OAuth: Users authenticate on Private Packagist with their Bitbucket Server accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/bitbucket-server-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps users and access permissions in sync with your Bitbucket Server projects
+    * Individual collaborators aren't supported
+* Code Credentials: personal access token which are available since Bitbucket Server 5.5 or username and password for older versions
+* Webhooks: Code changes and releases
+
+#### GitLab
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts
+* Synchronization:
+    * Keeps teams, their members, and access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't supported
+* Code Credentials: GitLab API token
+* Webhooks: Code changes and releases
+
+#### GitLab (Self-hosted)
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/gitlab-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps teams, their members, and access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't yet supported
+* Code Credentials: GitLab API Token
+* Webhooks: Code changes and releases
+
+#### AWS CodeCommit
+* Webhook:
+    * Users need to create a subscription using the generic hooks URL (from the package page) and send an [Amazon SNS message](https://docs.aws.amazon.com/sns/latest/dg/sns-message-and-json-formats.html)
```

Comment:
This is definitely clearer now. Instead of the SNS message I would maybe link to the Create an AWS CodeCommit Trigger docs.
```suggestion
    * Users need to [create an AWS CodeCommit Trigger](https://docs.aws.amazon.com/codecommit/latest/userguide/how-to-notify-sns.html) using the generic hooks URL from the package page
```